extensions/wikia/CommunityHeader/Navigation.class.php

```diff
@@ -27,14 +27,28 @@ public function __construct( DesignSystemCommunityHeaderModel $model, $wikiText
 
 	private function getExploreItems(): array {
 		$exploreItems = $this->model->getExploreMenu()['items'];
-
 		return array_map(
 			function ( $item ) {
-				return new Link(
-					new Label( $item['title']['key'], Label::TYPE_TRANSLATABLE_TEXT ),
-					$item['href'],
-					$item['tracking_label']
-				);
+				if ( array_key_exists( 'items', $item ) && !empty( $item['items'] ) ) {
```

Comment:
This would be a good use case for a recursive function

Reply:
going to leave as is for now.